src/amp.js

```diff
@@ -183,3 +183,5 @@ if (shouldMainBootstrapRun) {
     internalRuntimeVersion()
   );
 }
+
+console./*OK*/ log('trigger bundle size');
```

Comment:
Assuming this is unintentional?

Reply:
@kristoferbaxter sorry this was just to trigger an actual build test. will remove since it was tested

Reply:
Reminder to get rid of this line.